atsamd-hal-macros/src/lib.rs

```diff
@@ -61,7 +61,7 @@ fn hal_cfg_impl(args: TokenStream) -> Result<Group, Error> {
     Ok(cfgs)
 }
 
-/// Macro which expands to a `mod foo;` item with different paths for each
+/// Macro, which expands to a `mod foo;` item with different paths for each
```

Comment:
I think if we're going to bother with very small edits like this, it's worth knocking the two-line summary down to one line.